### PR TITLE
Consistency check done between README and mockingcase.js

### PR DESCRIPTION
**Pull request by Archulan**
@strdr4605 Sorry for the delay. I am stuck with work. I have done the requested changes.

### CODE REVIEW

File: src/mockingcase.js [hunk 1]
```diff
@@ -140,9 +142,9 @@ function isArrayOfStrings(input) {
 
   input.forEach((value, i) => {
     if (typeof value !== "string") {
-      throw TypeError(`Expected array of strings but got type '${typeof value}' at index ${i}`);
-    }
-  });
+    throw TypeError(`Expected array of strings but got type '${typeof value}' at index ${i}`);
+  }
+});
```
Comment (strdr4605) [inline src/mockingcase.js]: Don't remove the indentation

File: src/mockingcase.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
 /**
  * Converts the input string(s) to mOcKiNgCaSe.
  * @param {(string | string[])} input String(s) to be converted.
- * @param {object} [options={random: false,  onlyLetters: false}] Conversion options.
+ * @param {object} [options={random: false,  onlyLetters: false, firstUpper: false}] Conversion options.
```
Comment (strdr4605) [inline src/mockingcase.js]: Can you please add `upper` and `lower` to the options object?!

File: src/mockingcase.js [hunk 3]
```diff
@@ -64,10 +64,12 @@ function mOcKiNgCaSe(input = "", options) {
 /**
  * Outputs a message to the console in mOcKiNgCaSe.
  * @param {(string | string[])} input String(S) to be converted.
- * @param {object} [options={random: false}] Options for converting.
+ * @param {object} [options={random: false, onlyLetters: false, firstUpper: false}] Conversion options
```
Comment (strdr4605) [inline src/mockingcase.js]: add `upper` and `lower`

File: src/mockingcase.js [hunk 4]
```diff
@@ -92,12 +94,12 @@ mOcKiNgCaSe.config = defaultOptions => (input = "", overridedDefaultOptions) =>
 mOcKiNgCaSe.overrideString = () => {
   /**
    * Converts this string to mOcKiNgCaSe.
-   * @param {object} [options={random: false}] Options for converting.
+   * @param {object} [options={random: false, onlyLetters: false, firstUpper: false}] Options for converting.
```
Comment (strdr4605) [inline src/mockingcase.js]: add `upper` and `lower`

File: src/mockingcase.js [hunk 5]
```diff
@@ -111,12 +113,12 @@ mOcKiNgCaSe.overrideString = () => {
 
 /**
  * Overrides console.log input to print the input mOcKiNgCaSe.
- * @param {object} [options={random: false}] Options for converting.
+ * @param {object} [options={random: false, onlyLetters: false, firstUpper: false}] Options for converting.
```
Comment (strdr4605) [inline src/mockingcase.js]: add `upper` and `lower`

Comment (Archulan) [reply src/mockingcase.js]: Okay

**Pull request closed without merging**